api/graphql/makeModels.js

```diff
@@ -42,8 +42,19 @@ export default function makeModels (userId, isAdmin) {
     users: {
       typename: 'Person',
       model: User,
-      attributes: ['id', 'name', 'avatar_url', 'banner_url'],
-      relations: ['posts'],
+      attributes: [
+        'id',
+        'name',
+        'avatar_url',
+        'banner_url',
+        'bio',
+        'twitter_name',
+        'linkedin_url',
+        'facebook_url',
+        'url',
+        'location'
+      ],
+      relations: ['memberships', 'posts'],
```

Comment:
If we do `memberships` instead of `communities`, we have to add a filter for Membership, basically:

```js
filter: nonAdminFilter(q => {
  q.where('communities_users.community_id', 'in', myCommunityIds())
})
```

Otherwise we'd be exposing memberships to communities that the viewer is not allowed to know about.